modules/EnsEMBL/Web/Component/Variation/Phenotype.pm

```diff
@@ -243,6 +256,28 @@ sub table_data {
       $stats_values = '-';
     }
 
+    if ($allele) {
+      my $has_freq = 0;
+      my $allele_title = '';
+      foreach my $pop_name (sort { ($a !~ /ALL/ cmp $b !~ /ALL/) || $a cmp $b } keys %$pop_freq) {
+        if ($pop_freq->{$pop_name}{'freq'}{$allele}) {
+          my $freq = $pop_freq->{$pop_name}{'freq'}{$allele};
+          my $group = $pop_freq->{$pop_name}{'group'};
+          $allele_title .= ($allele_title eq '') ? "$group population frequencies for the allele &lt;b&gt;$allele&lt;/b&gt;:&lt;ul&gt;" : '';
+          $allele_title .= $lt.'li'.$gt.$pop_freq->{$pop_name}{'desc'};
+          $allele_title .= ' ('.$pop_freq->{$pop_name}{'label'}.')' if ($pop_freq->{$pop_name}{'desc'} ne $pop_freq->{$pop_name}{'label'});
+          $allele_title .= ' = ';
+          $allele_title .= ($freq > 0.01) ? $freq : $lt.qq{span style="color:#D00"$gt$freq$lt/span$gt};
+          $allele_title .= $lt.'/li'.$gt;
+        }
+      }
+      if ($allele_title ne '') {
+        $allele_title .= $lt.'/ul'.$gt;
+        $allele = qq{<span class="ht _ht"><span class="_ht_tip">$allele_title</span>$allele</span>};
```

Comment:
Instead of manually encoding the HTML, please use encode_entities from HTML::Entities.
Example: https://github.com/Ensembl/ensembl-webcode/blob/master/modules/EnsEMBL/Web/Component.pm#L427